www/src/components/StickyResponsiveSidebar/StickyResponsiveSidebar.js

```diff
@@ -50,12 +50,80 @@ class StickyResponsiveSidebar extends Component {
       open: false,
     };
     this.toggleOpen = this._toggleOpen.bind(this);
+    this.handleDocumentTouchMove = this._handleDocumentTouchMove.bind(this);
+    this.handleSidebarTouchStart = this._handleSidebarTouchStart.bind(this);
+  }
+
+  componentDidMount() {
+    document.addEventListener('touchmove', this.handleDocumentTouchMove);
+    document
+      .querySelector('.scrollable')
+      .addEventListener('touchstart', this.handleSidebarTouchStart);
```

Comment:
Strong initial dislike to all of the global handlers/selectors here. If we have to do this, I think we should wrap it up in a separate component rather than mix it into the sticky component.

Reply:
It might be possible to make it a bit more 'React' ish. For now we could start by adding a comment and linking to the blog post that describes this fix for the iOS overscroll bug. (http://blog.christoffer.online/2015-06-10-six-things-i-learnt-about-ios-rubberband-overflow-scrolling/)